<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addRectangleDialog</class>
 <widget class="QDialog" name="addRectangleDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>258</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>220</width>
    <height>258</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>220</width>
    <height>258</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Rectangle</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/rectangle.png</normaloff>:/rectangle.png</iconset>
  </property>
  <property name="toolTip">
   <string>Add Rectangle</string>
  </property>
  <property name="statusTip">
   <string>Add Rectangle</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>181</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="centerXLabel">
      <property name="text">
       <string>Center X (mm)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="centerXLineEdit">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>21</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Center X</string>
      </property>
      <property name="statusTip">
       <string>Center X</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="centerYLabel">
      <property name="text">
       <string>Center Y (mm)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="centerYLineEdit">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Center Y</string>
      </property>
      <property name="statusTip">
       <string>Center Y</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="widthLabel">
      <property name="text">
       <string>Width (mm)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="widthLineEdit">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Width</string>
      </property>
      <property name="statusTip">
       <string>Width</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="heightLabel">
      <property name="toolTip">
       <string>Add Rectangle</string>
      </property>
      <property name="statusTip">
       <string>Add Rectangle</string>
      </property>
      <property name="text">
       <string>Height (mm)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="heightLineEdit">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Height</string>
      </property>
      <property name="statusTip">
       <string>Height</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="angleLabel">
      <property name="text">
       <string>Angle</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="angleLineEdit">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Angle</string>
      </property>
      <property name="statusTip">
       <string>Angle</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="labelLabel">
      <property name="text">
       <string>Label</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="labelSpinBox">
      <property name="toolTip">
       <string>Label</string>
      </property>
      <property name="statusTip">
       <string>Label</string>
      </property>
      <property name="maximum">
       <number>240</number>
      </property>
      <property name="singleStep">
       <number>40</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addRectangleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addRectangleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
